### ISSUE TITLE
It says at line 35 that i have to report this to the developer

### ISSUE THREAD

**Issue by leonard-new** [ISeal-plugin-developement/SealLib#1]
Gist link:

https://gist.github.com/leonard-new/44f59c2e913649fe6faa70825fcdcc5f

**Comment by iseal56** [comment]
Are you sure the server is connected to internet?

**Comment by leonard-new** [comment]
yes beacuse at the same time my friends were joining but i didnt show that part of the log so i dont show their names and ips, and
once i was shutting down the server this part here https://gist.github.com/leonard-new/7498e34d15fa01e8ba6b149887a8caba
also said i needed to report it
